requirements.txt

```diff
@@ -1,3 +1,4 @@
 streamlit
 Unidecode
 streamlit-keyup>=0.1.6
+st-annotated-text
```

Comment:
Is this file needed? Given we have Pipfile?
If yes it's missing the streamlit-embed-code right?

Reply:
Yeah, we probably don't need it.